readme.md

```diff
@@ -1,18 +1,49 @@
 # Spring PetClinic Sample application [![setup automated](https://img.shields.io/badge/Gitpod-ready_to_code-orange?logo=gitpod)](https://gitpod.io/from-referrer/)
 
-Start coding in a [ready-to-code development environment](https://www.gitpod.io):
+[Petclinic](https://github.com/spring-projects/spring-petclinic) is a [Spring Boot](https://spring.io/guides/gs/spring-boot/) application built using [Maven](https://spring.io/guides/gs/maven/).
 
-<a href="https://gitpod.io/from-referrer/" style="padding: 10px;">
-    <img src="https://gitpod.io/button/open-in-gitpod.svg" width="150" alt="Push" align="center">
-</a>
```

Comment:
**Question:** Do we want to move this CTA down, seems logical to keep this up top as the big CTA? 

WDYT?